bindata/network/ovn-kubernetes/controller.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: ovn-kubernetes
+  name: sdn-controller-config
+data:
+  controller-config.yaml: |-
+{{.NetworkControllerConfig | indent 4}}
+
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: sdn-controller
+  namespace: ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      This deployment runs the openshift SDN networking controller.
```

Comment:
This is the SDN equivalent of `ovnkube --net-controller`

Reply:
Deleting the file.